pipenv/cli.py

```diff
@@ -1988,7 +1988,12 @@ def graph(bare=False, json=False):
             else:
                 click.echo(crayons.white(line, bold=False))
     else:
-        click.echo(c.out)
+        dep_graph = json.loads(c.out)
+        # Clean top level of dependency graph
+        dep_graph = [p for p in dep_graph if p['package']['package_name'] not in BAD_PACKAGES]
```

Comment:
Isn't this identical to just calling remove_json_packages? We could do that and reduce the likelihood of these two code chunks getting out of sync.

Reply:
No it isn't.  `remove_json_packages`, perhaps misnamed, only crawls the dependency tree.  It didn't remove top level packages.